src/reducers/versions.tsx

```diff
@@ -776,8 +818,13 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
       const { versionId } = action.payload;
 
       const version = state.versionInfo[versionId];
-      const { entries } = version;
-      const expandedPaths = entries
+
+      if (!version) {
+        _log.error(`Version missing for versionId: ${versionId}`);
```

Comment:
I wonder if this should throw an actual error, rather than just logging. It shouldn't really be possible to have called `expandedTree` without having a corresponding version loaded.